### DISCUSSION TITLE
[@types/node-schedule] Job.pendingInvocations

### DISCUSSION THREAD

**Discussion by wiggisser** [DefinitelyTyped/DefinitelyTyped#67898] [Issues with a @types package]
`Job.pendingInvocations` is not a function but just an array `Invocations[]`. This changed from v1.x to v2.x of `node-schedule`

See https://github.com/node-schedule/node-schedule/blob/c5a4d9a0dbcd5bda4996e089817e5669b5acd95f/lib/Job.js#L26

[a comment by an automated account is not shown]

**Comment by ghost** [comment]
Done in PR #68447
